dnf/cli/progress.py

```diff
@@ -111,16 +112,29 @@ def _update(self, now):
         msg = ' %5sB/s | %5sB %9s ETA\r' % (
             format_number(self.rate) if self.rate else '---  ',
             format_number(self.done_size),
-            format_time((self.total_size - self.done_size) / self.rate) if self.rate else '--:--')
+            format_time(
+                (self.total_size - self.done_size) / self.rate) if self.rate
+                                                                   and self.total_size else '--:--')
```

Comment:
I really don't know what is correct format here. Please @ignatenkobrain or @jsilhan do you know what would be correct line splitting in this case?